src/main.cpp

```diff
@@ -119,33 +120,14 @@ MM2PtrHook<void (*)(int, LPCSTR, char *)>
     !! THESE ARE ABSOLUTELY CRITICAL TO THE HOOK WORKING PROPERLY !!
 */
 
-MM2PtrHook<DWORD> $__VtResumeSampling       ( 0x5C86B8, 0x5DF710, 0x5E0CC4 );
-MM2PtrHook<DWORD> $__VtPauseSampling        ( 0x5C86C8, 0x5DF724, 0x5E0CD8 );
-MM2PtrHook<BOOL> $gameClosing               ( 0x667DEC, 0x6B0150, 0x6B1708 );
+MM2PtrHook<void(__stdcall*)(void)> $__VtResumeSampling  ( 0x5C86B8, 0x5DF710, 0x5E0CC4 );
```

Comment:
Technically the `__stdcall` is redundant since it has no parameters and doesn't need to clean up the stack.